rsopt/pkcli/pack.py

```diff
@@ -114,4 +130,3 @@ def configuration(config, ignore=None, add=None):
     tar_name = os.path.splitext(config)[0]
     create_filename = _create_tar(tar_name, file_list)
     print(f"Created tarball: {create_filename}")
-
```

Comment:
Purely style -- but I always leave a blank line. This could start a style war though.

Reply:
If one blank line is good how do you feel about two?

Reply:
It is actually for a reason related to unix tools and how some tools parse text files. But until you run into the issue it doesn't really make a difference

Reply:
For python alone, it is arguably style....but for other stuff, it isn't. I always put them in to my files unless it is purely a file to be `include`d in another.

pylint reports a warning (https://stackoverflow.com/questions/44317765/pylint-giving-me-final-new-line-missing), c++ requires it by standard, etc. But if someone is running only on Windows I can't really say they require it outside of style